runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonConfig.java

```diff
@@ -81,4 +81,24 @@ default boolean showCannonSpots()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "ammoAmount",
+		name = "Ammo left",
+		description = "Configure to set the amount of ammo left to receive ammo left notification"
+	)
+	default int ammoAmount()
+	{
+		return 5;
+	}
+
+	@ConfigItem(
+		keyName = "notifyAmmoLeft",
+		name = "Ammo left notification",
+		description = "Sends a notification when cannon ammo is under the specified amount"
+	)
+	default boolean notifyAmmoLeft()
```

Comment:
This should probably be removed. Other configurations like this one are set up to use `0` as an "off" value. This removes the need for a second configuration value.